module/Database/src/Mapper/Member.php

```diff
@@ -59,15 +59,17 @@ public function search(string $query): array
             ->from(MemberModel::class, 'm')
             ->where("CONCAT(LOWER(m.firstName), ' ', LOWER(m.lastName)) LIKE :name")
             ->orWhere("CONCAT(LOWER(m.firstName), ' ', LOWER(m.middleName), ' ', LOWER(m.lastName)) LIKE :name")
+            ->orWhere("m.email LIKE :name")
```

Comment:
I think this can potentially lead to less accurate results, especially for actual members (who may have updated their e-mail address to not be their student e-mail address).

Reply:
Do you think it would be better to only do this when it is a valid email address? Because I do remember there were cases where members had to be searched for by their email address (e.g. when someone wanted to register for a mailing list without providing their name) and this would simplify that. 

```suggestion

            ->setMaxResults(32)
            ->orderBy('m.lidnr', 'DESC')
            ->setFirstResult(0);
            
        if (filter_var($query, FILTER_VALIDATE_EMAIL)) {
            $qb->orWhere("m.email LIKE :name");
        }
```

Reply:
I did the above in cead09482498abd8002ed8f3db3dc981592ca435